<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Оценка случайности</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Алгоритмические значения</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Табличные значения</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="alg_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>0..9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10..99</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>100..999</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_table">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>321</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>0..9</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>10..99</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>100..999</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>435</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Мера случайности</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_alg_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_1">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_table_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="fill">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>465</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Заполнить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>496</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Ручной ввод:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manual_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>526</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>546</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Мера случайности:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="meas_manual">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>550</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
